source/frontend/src/features/mapSideBar/research/add/AddResearchContainer.tsx

```diff
@@ -199,7 +199,7 @@ export const AddResearchContainer: React.FunctionComponent<IAddResearchContainer
       {formikProps => (
         <MapSideBarLayout
           title="Create Research File"
-          icon={<MdTopic title="User Profile" size="2.5rem" className="mr-2" />}
+          icon={<MdTopic title="User Profile" size="2.5rem" />}
```

Comment:
nit: all other icons are sized at 2.6 rem - would it make sense to adjust this one too?